src/panel/json.js

```diff
@@ -62,7 +62,7 @@ export default class Code extends React.Component {
     } else if (err.length) {
       this.handleGeoJSONError(editor, err);
     }
-    if (!(err instanceof Error || err.every(e => e.level !== 'message'))) {
+    if (!(err instanceof Error || !err.every(e => !e.message))) {
```

Comment:
This doesn't seem like the right fix - it's asserting about the `message` property, which is set on all errors, right?

Reply:
There is still a problem with the existing check, when `err` resolves to empty array ([]) it will satisfy the every clause so the whole check will fail. 

Suggested fix: 

     if (!(err instanceof Error || err.length || err.every(e => e.level !== 'message')))